packages/esm-patient-forms-app/src/config-schema.ts

```diff
@@ -1,6 +1,35 @@
 import { validator, Type } from '@openmrs/esm-framework';
 
 export const configSchema = {
+  formsSectionsConfig: {
+    _type: Type.Array,
+    _elements: {
+      name: {
+        _type: Type.String,
+        _description: 'Name of the section.',
+      },
+      labelCode: {
+        _type: Type.String,
+        _description: 'Label to be used to translate the section.',
+      },
+      forms: {
+        _type: Type.Array,
+        _description:
+          'List of forms that should be associated with this section, it can be used the `Form UUID` or the `Encounter Type UUID` if you have both the `Form UUID` will be used.',
+        _elements: {
+          formUuid: {
+            _type: Type.String,
+            _description: 'Uuid of the form.',
+          },
+          encounterTypeUuid: {
+            _type: Type.String,
+            _description: 'Uuid of the encounter type.',
+          },
+        },
+      },
+    },
+    _default: undefined,
```

Comment:
I don't think the config system works really well with default `undefined`. Better to just use an empty array (which is still falsy in JS).

```suggestion
    _default:[],
```